holidays/countries/sao_tome_and_principe.py

```diff
@@ -0,0 +1,111 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.groups import ChristianHolidays, InternationalHolidays
+from holidays.observed_holiday_base import ObservedHolidayBase, SUN_TO_NEXT_MON, SAT_TO_PREV_FRI
+
+
+class SaoTomeAndPrincipe(ObservedHolidayBase, ChristianHolidays, InternationalHolidays):
+    """São Tomé and Príncipe holidays.
+
+    References:
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_São_Tomé_and_Príncipe>
```

Comment:
Aside from perhaps adding ST's government sources to the list - we're more or less done now I think? 👍 
```suggestion
        * <https://web.archive.org/web/20190319201800/http://www.mnec.gov.st/index.php/o-pais?showall=1>
        * <https://visitsaotomeprincipe.st/pt/informacoes-uteis/feriados>
        * <https://en.wikipedia.org/wiki/Public_holidays_in_São_Tomé_and_Príncipe>
```